components/renku_data_services/connected_services/api.spec.yaml

```diff
@@ -109,6 +109,35 @@ paths:
           $ref: "#/components/responses/Error"
       tags:
         - oauth2
+  /oauth2/providers/{provider_id}/simple_connect:
```

Comment:
As a general remark: I think this API endpoint should be relocated somewhere else. It should not have `oauth2` in the path when this is to handle third-party platforms which DO NOT implement OAuth 2.0.

Some suggestions: `/integrations/...`, `/non-oauth2/...`, `/providers/...`, etc.